src/style.css

```
@@ -1,3 +1,82 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:500,700");
+
 body {
   margin: 0;
+  background-color: rgb(120, 120, 255);
+  padding: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", sans-serif;
+
+}
+
+nav {
```

```suggestion
.nav {
```
better to use class selectors

---

- [DEMO LINK](https://DavaiBezDava1.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://DavaiBezDava1.github.io/layout_moyo-header/report/html_report/)